frontend/apps/core-app/App.tsx

```diff
@@ -25,10 +26,31 @@ const App: React.FC = () => {
     Roboto_400Regular_Italic,
     Roboto_700Bold,
   });
+
+  const linking = {
+    prefixes: [
+      `https://${Constants.manifest?.scheme}.com`,
+      `${Constants.manifest?.scheme}://`,
+    ],
+    config: {
+      screens: {
+        Recipients: 'recipients',
+        RecipientList: '/',
+        RecipientRegistration: 'recipients/register',
+        RecipientProfile: {
+          path: '/:id',
+          parse: {
+            id: String,
+          },
+        },
+      },
+    },
+  };
```

Comment:
Maybe this should live in it's own file. It'll likely grow quite large to be in `App.tsx`